### DISCUSSION TITLE
Physics

### DISCUSSION THREAD

**Discussion by JackCrumpLeys** [DigitalExtinction/Game#445] [Ideas]
This may already be the plan but could we have simulated physics for projectiles which can be dodged by AI and can miss. this would make the game-play more immersive and add more opportunity for interesting unit choices.

have a look at [Zero-k](https://zero-k.info/).

**Comment by Indy2222** [comment]
I agree, it is the plan that projectile ballistics should be simulated + aiming should not be 100% precise. OTOH the simulation must be performance optimized since there might be hundred / thousands of flying projectiles at a given moment.
